### DISCUSSION TITLE
Movies being ignored once scraped

### DISCUSSION THREAD

**Discussion by Robert106** [itsToggle/plex_debrid#48] [Q&A] [answered]
Hi, I hope someone can help with this issue :)

I'm trying to set this up on my Windows PC and all seemed to be going well... I followed the recommended steps in the README and it was all working until the scraping part. When I add something to my watchlist on Plex the sources are found but ignored. I'm not sure if I've missed something but any help would be appreciated (image below for reference). 
Thanks a lot

<img width="591" alt="image" src="https://user-images.githubusercontent.com/59128322/181382581-d1ca5493-9b28-4d1a-972f-71dfd8907e29.png">

**Comment by itsToggle** [comment, marked as answer]
Hi! Which debrid service have you set up? This looks similar to an old issue, where the user set up realdebrid but did not have a realdebrid premium account. Is this the case for you as well? A premium account is needed. If you decided to get premium, consider using my realdebrid [affiliate link](http://real-debrid.com/?id=5708990) :)

To find more information on the error, you can also turn on debug printing by navigating to "/Settings/Ui Settings/Debug printing".

**Reply by Robert106** [reply]
Ah I hadn't realised premium was needed. Thanks so much for your help!
